src/pykx/pandas_api/pandas_meta.py

```diff
@@ -257,6 +257,14 @@ def sum(self, axis=0, skipna=True, numeric_only=False, min_count=0):
             min_count
         ), cols)
 
+    @convert_result
+    def nunique(self, axis=0, dropna=True):
+        res, cols = preparse_computations(self, axis, skipna=False)
+        filterNan = q('{$[all[10h=type each x]|11h = type x;x;'
```

Comment:
Overall it's a great job, just one minor thing. They normally don't use CamelCase convention for variable naming, maybe I would consider changing it to simply `filternan`

Reply:
done